cmake/FindLibClang.cmake

```diff
@@ -48,7 +50,7 @@ endif(NOT LLVM_FOUND)
 
 find_path(
   LIBCLANG_INCLUDE_DIRS clang-c/Index.h
-  PATHS ${LLVM_INCLUDE_DIR}
+  PATHS ${LLVM_INCLUDE_DIR} /usr/include
```

Comment:
why?

Reply:
LLVM_INCLUDE_DIR wasn't enough to find LLVM-18 on manjaro linux latest. can be removed ofc

Reply:
I was more curious to know if we can have it already included in that variable a bit earlier